docs/vis_nir_tutorial.md

```diff
@@ -345,21 +344,7 @@ Write co-result data out to a file.
 
 ```python
 
-    coresult=open(args.coresult,"a")
-    coresult.write('\t'.join(map(str,nhist_header)))
-    coresult.write("\n")
-    coresult.write('\t'.join(map(str,nhist_data)))
-    coresult.write("\n")
-    for row in nir_imgs:
-      coresult.write('\t'.join(map(str,row)))
-      coresult.write("\n")
-    coresult.write('\t'.join(map(str,nshape_header)))
-    coresult.write("\n")
-    coresult.write('\t'.join(map(str,nshape_data)))
-    coresult.write("\n")
-    coresult.write('\t'.join(map(str,nir_imgs[0])))
-    coresult.write("\n")
-    coresult.close()
+    pcv.print_result(filename=args.coresult)
```

Comment:
I just realized this will be a problem because the VIS measurements will still be in the output measurements. The NIR measurements will overwrite some but not all unless the user reruns all the same analyses, which we usually don't.

I'm thinking we could add a `clear` method that would delete all current measurements. That would be called by `pcv.outputs.clear()`.